javascript-timers/timeout.js

```diff
@@ -0,0 +1,5 @@
+const message = document.querySelector('.message');
+
+setTimeout(() => {
+  message.textContent = 'Hello There';
+}, 5000);
```

Comment:
This isn't a two-second delay:

---

![image](https://user-images.githubusercontent.com/16143771/168209876-7c85e1f5-c8e1-4047-9faa-bcc4b5dc4730.png)